docs/configuring-playbook-bot-mjolnir.md

```diff
@@ -29,7 +29,7 @@ Refer to the documentation on [how to obtain an access token](obtaining-access-t
 
 ## 3. Make sure the account is free from rate limiting
 
-You will need to prevent Synapse from rate limiting the bot's account. This is not an optional step. If you do not do this step Mjolnir will crash. [Currently there is no Synapse config option for this](https://github.com/matrix-org/synapse/issues/6286) so you have to manually edit the Synapse database. Manually editing the Synapse database is rarely a good idea but in this case it is required. Please ask for help if you are uncomfortable with these steps.
+You will need to prevent Synapse from rate limiting the bot's account. This is not an optional step. If you do not do this step Mjolnir will crash. There is a Synapse config option for this but this can also be manually done by editing the Synapse database. Manually editing the Synapse database is rarely a good idea. Please ask for help if you are uncomfortable with these steps.
```

Comment:
Saying "there is a config option" is better than not mentioning it, but it's even better if we link to the option, etc.

Reply:
Does [this](https://github.com/matrix-org/synapse/blob/f9f03426de338ae1879e174f63adf698bbfc3a4b/docs/admin_api/user_admin_api.md#override-ratelimiting-for-users) help? I'm not too sure what to include...

Reply:
Yes, sounds good